python/foglamp/tasks/purge/purge.py

```diff
@@ -64,28 +63,29 @@ class Purge(FoglampProcess):
     _CONFIG_CATEGORY_NAME = 'PURGE_READ'
     _CONFIG_CATEGORY_DESCRIPTION = 'Purge the readings table'
 
-    def __init__(self):
+    def __init__(self, loop=None):
         super().__init__()
         self._logger = logger.setup("Data Purge")
         self._audit = AuditLogger(self._storage)
+        if loop is None:
+            self.loop = asyncio.get_event_loop()
+        else:
+            self.loop = loop
```

Comment:
`self.loop = asyncio.get_event_loop() if loop is None else loop`

Reply:
trivial